lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -2727,14 +2727,175 @@ def self.notify
+          :description => 'If `true`, disables RXSS (reflected cross-site scripting) detection'
+        },
+        :'security.scan_schedule.delay' => {
+          :default => 0,
+          :public => true,
+          :type => Integer,
+          :allowed_from_server => true,
+          :description => 'The delay field indicated time in minutes before the IAST scan starts after the application starts'
+        },
+        :'security.scan_schedule.duration' => {
+          :default => 0,
           :public => true,
           :type => Integer,
+          :allowed_from_server => true,
+          :description => 'The duration field specifies the duration of the IAST scan in minutes. This determines how long the scan will run'
+        },
+        :'security.scan_schedule.schedule' => {
+          :default => '',
+          :public => true,
+          :type => String,
+          :allowed_from_server => true,
+          :description => 'The schedule field specifies a cron expression that defines when the IAST scan should run',
+          :dynamic_name => true
+        },
+        :'security.scan_schedule.always_sample_traces' => {
+          :default => false,
           :external => true,
+          :public => true,
+          :type => Boolean,
           :allowed_from_server => false,
-          :description => 'Defines the request body limit to process in security events (in KB). The default value is 300, for 300KB.'
+          :description => 'always_sample_traces permits IAST to actively gather trace data in the background, and the collected data will be used by Security Agent to perform an IAST Scan at the scheduled time'
+        },
+        :'security.scan_controllers.iast_scan_request_rate_limit' => {
+          :default => 3600,
+          :public => true,
+          :type => Integer,
+          :allowed_from_server => true,
+          :description => 'Number of http request limit for ISAT scan per minute. Default is 3600. Minimum is 12 and maximum is 3600'
+        },
+        :'security.scan_controllers.scan_instance_count' => {
+          :default => 0,
+          :public => true,
+          :type => Integer,
+          :allowed_from_server => true,
+          :description => 'Scan instance count controls the number of application instances for a specific entity where IAST analysis is performed'
+        },
+        :'security.scan_controllers.report_http_response_body' => {
+          :default => true,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => true,
+          :description => 'report_http_response_body allows user to enable/disable sending of HTTP responses body, Disabling this would also disable RXSS vulnerability detection'
+        },
+        :'security.iast_test_identifier' => {
+          :default => nil,
+          :public => true,
+          :type => String,
+          :allowed_from_server => true,
+          :description => 'This will allow users to run IAST for CI/CD'
```

Comment:
I'm not sure what an example input would be here- is there any way to expand on the description?

Reply:
@hannahramadan Updated description in [this](https://github.com/newrelic/newrelic-ruby-agent/pull/2934/commits/4204fa5f3f692cd8bb28c69f082a3f9d345ba30b) commit